spec/forms/claimant_form_spec.rb

```diff
@@ -151,6 +151,14 @@
       expect(claimant_form.errors).not_to include :mobile_number
     end
 
+    it 'disallows an invalid email' do
+      claimant_form.contact_preference = 'email'
+      claimant_form.email_address = 'jls435829@gmail..com'
```

Comment:
Im not sure if this is the email address that I gave you as an example ?  If it is, please change it to something else as that was a real user.  If not, ignore me :-)